tests/regression/77-lin2vareq/28-narrowing-on-steroids.c

```diff
@@ -0,0 +1,29 @@
+// SKIP PARAM: --set ana.activated[+] lin2vareq --enable ana.int.interval --set solver slr3tp
```

Comment:
What's with the unusual solver?

Reply:
This test is used to check if a `meet` on `narrow` has an impact on the precision of our analysis and the `narrow` is not executed with the usual solver, but with this specific solver. We got this solver from previous test cases which tested similar behavior.